config.yaml

```diff
@@ -71,9 +71,11 @@ languages:
         url: https://scipy.github.io/devdocs/index.html
       - title: Download
         url: /download
-      - title: Mailing Lists
+      - title: Community
         url: /mailing-lists
```

Comment:
This file should be moved to `community.md` I'd say

Reply:
I agree, I had open #426 and will open a follow up PR soon.